packages/react/src/components/numberInput/NumberInput.tsx

```diff
@@ -117,6 +118,29 @@ export const NumberInput = React.forwardRef<HTMLInputElement, NumberInputProps>(
     const notifyScreenReaderStepperChangedValue = () => {
       setScreenReaderValue(String(inputRef.current.value));
     };
+    let throttledMouseWheel = false;
```

Comment:
Noice 👍 Could this be written as a custom hook for cleaner syntax? 🤔  E.g. `useThrottledMouseWeel(inputRef)`

Reply:
Will check & test 👍